audio_service/android/src/main/java/com/ryanheise/audioservice/AudioServicePlugin.java

```diff
@@ -724,7 +702,6 @@ public void onSetRating(RatingCompat rating) {
         public void onSetRating(RatingCompat rating, Bundle extras) {
             invokeMethod("setRating", mapOf(
                         "rating", rating2raw(rating),
-                        // XXX: ? was extras.getSerializable("extrasMap")
```

Comment:
@hacker1024 I was meaning to ask you about this since line which I commented out since I couldn't find "extrasMap" referenced anywhere else. Let me know if my replacement code on the following line breaks your app.